core/src/main/java/org/apache/spark/util/collection/unsafe/sort/PrefixComparators.java

```diff
@@ -82,37 +81,37 @@ public static long computePrefix(double value) {
   //
 
   public static final class UnsignedPrefixComparator extends RadixSortSupport {
-    @Override public final boolean sortDescending() { return false; }
-    @Override public final boolean sortSigned() { return false; }
+    @Override public boolean sortDescending() { return false; }
+    @Override public boolean sortSigned() { return false; }
     @Override
-    public final int compare(long aPrefix, long bPrefix) {
+    public int compare(long aPrefix, long bPrefix) {
       return UnsignedLongs.compare(aPrefix, bPrefix);
     }
   }
 
   public static final class UnsignedPrefixComparatorDesc extends RadixSortSupport {
-    @Override public final boolean sortDescending() { return true; }
-    @Override public final boolean sortSigned() { return false; }
+    @Override public boolean sortDescending() { return true; }
+    @Override public boolean sortSigned() { return false; }
     @Override
-    public final int compare(long bPrefix, long aPrefix) {
+    public int compare(long bPrefix, long aPrefix) {
```

Comment:
Oh, it's definitely final. It's just `RedundantModifier` error since the class `SignedPrefixComparator` is already `final`.